software/contrib/menu.py

```diff
@@ -32,6 +32,7 @@
     ["Diagnostic",        "contrib.diagnostic.Diagnostic"],
     ["EnvelopeGen",       "contrib.envelope_generator.EnvelopeGenerator"],
     ["Euclid",            "contrib.euclid.EuclideanRhythms"],
+    ["G & T",             "contrib.gates_and_triggers.GatesAndTriggers"],
```

Comment:
[optional] We've got 16 characters, I think it'd be nice for discovery if the name in the menu was more descriptive. 'Gates & Triggers" fits.

Reply:
Done